crypto-test/.jvm/src/test/scala/org/bitcoins/crypto/ECPrivateKeyTest.scala

```diff
@@ -1,86 +1,17 @@
 package org.bitcoins.crypto
 
-import org.bitcoins.core.config.{MainNet, RegTest, SigNet, TestNet3}
-import org.bitcoins.core.crypto.ECPrivateKeyUtil
-import org.bitcoins.testkitcore.gen.{
-  ChainParamsGenerator,
-  CryptoGenerators,
-  NumberGenerator
-}
-import org.bitcoins.testkitcore.util.BitcoinSUnitTest
-
-class ECPrivateKeyTest extends BitcoinSUnitTest {
+class ECPrivateKeyTest extends BitcoinSCryptoTest {
```

Comment:
Is there a reason the other tests in this file can't be shared?

Reply:
To avoid circular dependencies between `crypto` and `core`